### ISSUE TITLE
Improved Installation/Setup Documentation

### ISSUE THREAD

**Issue by pcrossnz** [directus/docs#140]
Hi all,

I'm just providing some feedback. I've successfully been able to install various other headless CMS's, but have been unable to do so with Directus due to the difficulty in following the documentation. There's no doubt that I'm lacking some skills here, but it would be great if there was a walk-through for deployment that truly guides the user through the process. 

Thanks

**Comment by benhaynes** [comment]
Hey @pcrossnz — thank you for the honest feedback!

Could you describe where you're running into issues in a bit more detail? Our primary install method is just one `git` command and running the setup wizard. We have many other methods too... some which are even easier!

We're going to spend some serious time on Docs over the next month or two, so any direction you have would help!

https://docs.directus.io/getting-started/installation.html#setup

**Comment by pcrossnz** [comment]
Thank's Ben. Really appreciate the message. I  managed to get Directus via git no problems. However, with my limited experience, I did not know how to proceed with the following steps:

_Set your document root to the /public directory
Navigate your browser to the App at /admin

I did eventually manage to get directus installed via softaculous last night, which is a helpful tool. The CMS is amazing! Once again, my lack of experience makes it difficult for me to follow the API documentation at the moment. I will persevere though. I have found it extremely helpful with other APIs (such as strapi and squidex) to have the autogenerated documentation via swagger. 

Thanks again Ben,

Phil
